ci-operator/config/openshift/master-dns-operator/openshift-master-dns-operator-master.yaml

```diff
@@ -0,0 +1,49 @@
+base_images:
+  base:
+    cluster: https://api.ci.openshift.org
+    name: origin-v4.0
+    namespace: openshift
+    tag: base
+binary_build_commands: make build
+canonical_go_repository: github.com/openshift/master-dns-operator
+images:
+- dockerfile_path: Dockerfile
+  from: base
+  to: master-dns-operator
```

Comment:
We aren’t using the word “master” if we can avoid it.

Reply:
I don’t really think this is the right name for this.  Its too specific and confusing.

What other code will this live in?  Why isn’t this part of openshift-kube-apiserver-operator?

Reply:
If this is about etcd management then give a name like that and let’s fold the bootstrap aspects of setup-etcd-bootstrap into this or cluster-bootstrap.

Reply:
@smarterclayton We can definitely rename, though not sure that it makes sense to lump with the apiserver operator. This deals with cloud infrastructure and not just control plane stuff. The other cloud infra operator we have is the machine api operator but not sure it fits there either. How does `control-plane-dns-operator` sound?

Reply:
Ahh missed your last comment. Ack. Makes sense